Shared/Samples/Show WFS layer with XML query/ShowWFSLayerWithXMLQueryView.swift

```diff
@@ -0,0 +1,135 @@
+                }
+                .task {
+                    do {
+                        // Load the WFS data.
+                        try await loadData()
+                        if let extent = seattleTreesTable.extent, !isLoading {
+                            // Animate the map to the extent of the data over 1 second.
+                            await mapView.setViewpoint(
+                                Viewpoint(boundingGeometry: extent),
+                                duration: 1.0
+                            )
+                        }
+                    } catch {
+                        // If an error occurs during loading, capture it to trigger an alert.
+                        self.error = error
+                    }
+                }
+            // Display an alert if there is an error during data loading.
+                .errorAlert(presentingError: $error)
+        }
+    }
+    
+    /// Asynchronous function to load data from the WFS service
+    func loadData() async throws {
+        isLoading = true
+        // Set the axis order to not swap X and Y (used for coordinate systems.)
+        seattleTreesTable.axisOrder = .noSwap
+        // Use manual cache mode so data must be explicitly loaded from the service.
+        seattleTreesTable.featureRequestMode = .manualCache
+        try await seattleTreesTable.load()
+        // Create a feature layer from the table and add it to the map.
+        let layer = FeatureLayer(featureTable: seattleTreesTable)
+        map.addOperationalLayer(layer)
+        // Populate the feature table with data from the WFS service using an XML query.
+        _ = try await seattleTreesTable.populateFromService(
+            usingXMLRequest: .xmlQuery,
+            clearCache: true
+        )
+        // Mark that loading has completed.
+        isLoading = false
+    }
+    
+    var loadingView: some View {
+        ProgressView(
+            """
+            Loading query
+            data
+            """
+        )
+        .padding()
+        .background(.ultraThinMaterial)
+        .clipShape(.rect(cornerRadius: 10))
+        .shadow(radius: 50)
+        .multilineTextAlignment(.center)
+    }
+}
+
+#Preview {
+    ShowWFSLayerWithXMLQueryView()
+}
+
+private extension URL {
+    /// Static property to return the full URL for the WFS GetCapabilities request.
+    static var wfsUrl: URL {
+        URL(string: "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities")!
+    }
+}
+
+extension String {
+    /// This string matches the `typeNames` used in the XML query to identify the layer to query.
+    static var seattleTreesDowntown: String {
+        "Seattle_Downtown_Features:Trees"
+    }
+    
+    /// XML query to request features from the WFS service
+    /// This specific query fetches only tree features where the "SCIENTIFIC" field equals "Tilia cordata"
+    static let xmlQuery = """
```

Comment:
Can you change this to `static var` as well? Not a big deal but we try to limit the usage of [static data](https://icarus.cs.weber.edu/~dab/cs1410/textbook/4.Pointers/memory.html) in memory. See the contribution guideline for more details if you are interested.

#112